src/lib/components/button/NewProposalButton.tsx

```diff
@@ -1,18 +1,21 @@
 import { Button, Menu, MenuButton, MenuItem, MenuList } from "@chakra-ui/react";
 
 import { Tooltip } from "../Tooltip";
-import { useInternalNavigate } from "lib/app-provider";
+import { useInternalNavigate, useGovConfig } from "lib/app-provider";
 import { CustomIcon } from "lib/components/icon";
 import { AmpEvent, AmpTrack } from "lib/services/amplitude";
 // import { useGovParams } from "lib/services/proposalService";
 // import { AccessConfigPermission } from "lib/types";
 
 export const NewProposalButton = () => {
   const navigate = useInternalNavigate();
+  const govConfig = useGovConfig({ shouldRedirect: false });
   // const { data: govParams } = useGovParams();
   // const isPermissionless =
   //   govParams?.uploadAccess.permission === AccessConfigPermission.EVERYBODY;
 
+  if (govConfig.enabled && govConfig.disableOpenProposal) return null;
```

Comment:
can we just check for `govConfig.disableOpenProposal`?

Reply:
We can't because we use discriminated union.